plugins/woocommerce/client/admin/client/settings-payments/onboarding/providers/woopayments/steps/business-verification/components/form/index.tsx

```diff
@@ -0,0 +1,191 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { Button } from '@wordpress/components';
+import { isEmpty, mapValues } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { useOnboardingContext } from '../../../../data/onboarding-context';
+import { useStepperContext } from '../stepper';
+import { Item as SelectItem } from '../../../../components/custom-select-control';
+import { ListItem as GroupedSelectItem } from '../../../../components/grouped-select-control';
+import {
+	GroupedSelectField,
+	GroupedSelectFieldProps,
+	SelectField,
+	SelectFieldProps,
+	TextField,
+	TextFieldProps,
+} from './fields';
+import { useBusinessVerificationContext } from '../../data/business-verification-context';
+import { OnboardingFields } from '../../types';
+import { completeSubStep, isPreKYCComplete } from '../../utils';
```

Comment:
Nitpick: It looks like in other places in the code we use `Kyc` (e.g. `AccountKycSession` rather than `KYC` - what do you think about using `isPreKycComplete` instead here?

Reply:
Good catch, updated!